universum/submit.py

```diff
@@ -22,7 +22,9 @@ def define_arguments(parser):
                             help="Only submit existing vcs modifications, no adding or deleting")
 
         parser.add_argument('--commit-message', '-cm', dest='commit_message', metavar="COMMIT_MESSAGE",
-                            help='Commit message to add')
+                            help="Pass the commit message to add via environment variable, or redirect to stdin "
+                                 "by passing '-' as param value, or pass a file path to read the message from "
```

Comment:
`Pass the commit message to ... redirect to stdin by passing '-' as param value`
I think it's not entirely clear